README.md

```diff
@@ -1,7 +1,25 @@
-project-template
+Meteor Demo
 ==============================
 
-template for the team to use
+This repository has materials for a hands-on tutorial to demonstrate tools developed by Red Hat's Artificial Intelligence Center of Excellence.
+
+This tutorial highlights the capabilities of [Project Meteor](https://github.com/AICoE/meteor), which is a web application that converts users' GitHub repositories into [JupyterBook](https://jupyterbook.org/intro.html) environments.
```

Comment:
"... web application that converts users' GitHub repositories into both a [JupyterBook](https://jupyterbook.org/intro.html) as well as a deployment interactive environment.